### PR TITLE
增加主机模型内置属性支持unix类型

### PR DESCRIPTION
**Pull request by wjyfelen**
### 新加的功能:
- 增加主机模型内置属性支持unix类型

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-cmdb?pullRequest=5295) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-cmdb?pullRequest=5295) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-cmdb?pullRequest=5295) it.</sub>

File: src/scene_server/admin_server/upgrader/y3.9.202105251151/add_unix_property.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+	"configcenter/src/common"
+	"configcenter/src/common/condition"
+	"configcenter/src/common/mapstr"
+	"configcenter/src/common/metadata"
+	"configcenter/src/scene_server/admin_server/upgrader"
+	"configcenter/src/storage/dal"
+)
+
+type Attribute struct {
```
Comment (breezelxp) [inline src/scene_server/admin_server/upgrader/y3.9.202105251151/add_unix_property.go:26]: 小写

File: src/scene_server/admin_server/upgrader/y3.9.202105251151/add_unix_property.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	CreateTime        *time.Time  `json:"create_time" bson:"create_time"`
+	LastTime          *time.Time  `json:"last_time" bson:"last_time"`
+}
+
+func addUNIXProperty(ctx context.Context, db dal.RDB, conf *upgrader.Config) error {
+
+	cond := condition.CreateCondition()
+	cond.Field(common.BKOwnerIDField).Eq(common.BKDefaultOwnerID)
+	cond.Field(common.BKObjIDField).Eq(common.BKInnerObjIDHost)
+	cond.Field(common.BKPropertyIDField).Eq(common.BKOSTypeField)
```
Comment (breezelxp) [inline src/scene_server/admin_server/upgrader/y3.9.202105251151/add_unix_property.go:57]: 缺一个条件bk_biz_id:0

File: src/scene_server/admin_server/upgrader/y3.9.202105251151/add_unix_property.go [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+	ostypeProperty := Attribute{}
```
Comment (breezelxp) [inline src/scene_server/admin_server/upgrader/y3.9.202105251151/add_unix_property.go:59]: ostypeProperty := new(Attribute)

File: src/scene_server/admin_server/upgrader/y3.9.202105251151/add_unix_property.go [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	if err != nil {
+		return err
+	}
+
+	enumOpts, err := metadata.ParseEnumOption(ctx, ostypeProperty.Option)
+	if err != nil {
+		return err
+	}
+	for _, enum := range enumOpts {
+		if enum.ID == "4" {
```
Comment (breezelxp) [inline src/scene_server/admin_server/upgrader/y3.9.202105251151/add_unix_property.go:70]: HostOSTypeEnumUINX

File: src/scene_server/admin_server/upgrader/y3.9.202105251151/add_unix_property.go [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+			return nil
+		}
+	}
+
+	aixEnum := metadata.EnumVal{
+		ID:   "4",
```
Comment (breezelxp) [inline src/scene_server/admin_server/upgrader/y3.9.202105251151/add_unix_property.go:76]: HostOSTypeEnumUINX

**Review by breezelxp — CHANGES_REQUESTED** [review]

**Pull request closed without merging**